src/provisioner/provisioner.py

```diff
@@ -400,3 +405,16 @@ def _validate_provisioning_status(cortx_conf: MappedConf, node_id: str, apply_ph
         else:
             Log.info(msg)
         return validate_result[0], validate_result[1]
+    
+    @staticmethod
+    def is_cluster_healthy():
+        # Checks if cluster is healthy
+        CortxProvisioner._get_resource_health(resource="cortx")
+        return True
+
+    @staticmethod
+    def _get_resource_health(resource='cortx'):
```

Comment:
_get_resource_health is a generic function and must not have a default. 

```
def _get_resource_health(resource: str):
        """API call to get cluster health."""
        # TODO Make a call to HA Health API to get the resource status
        return "OK"
```

Reply:
done